active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+> See [proposal document](https://github.com/Microsoft/microsoft-ui-xaml/issues/304) for more background
+
+The TabView control is a way to display a set of tabs and their respective content. Tab controls are useful for displaying several pages (or documents) of content while giving a user the capability to rearrange, open, or close new tabs. 
+
+![Tabs in Edge](https://user-images.githubusercontent.com/25991996/52679758-f6d6ad80-2eea-11e9-9955-fd111c26d982.png)
+
+Tab-like UI comes in two distinct styles that affect not only the visualization of the control, but also the user experience:
+* Static tabs
+    * Generally display a set number of pages
+    * User can change between pages, but cannot open new pages, close pages, or rearrange pages
+* Document tabs
+    * Display a variable number of pages
+    * User can add, remove, and rearrange pages
+    * User can "tear off" tabs into their own window or "recombine" tabs from one group into another group
+
+The Xaml platform already provides several controls to represent "static-style tabs" including [Pivot](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/pivot) and top [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes). However, these controls do not well support "document-style" tabs due to a variety of limitations, including:
+* Significant effort up to (and including) retemplating to build simple tabs
+* No built-in support for closing tabs
+* No built-in support for drag/drop into new windows
+* No built-in support for moving a tab from a window and combining it with another window
+* Limited keyboard and accessibility support
+
+This spec therefore covers "document-style tabs" specifically as this platform gap isn't easily solved with an app-side workaround. 
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+TabView is a collection of tabs that each represents a new page or document in your app. Use TabView when your app has several pages of content and the user expects to be able to add, close, and rearrange the pages.
```

Comment:
> to be able to **switch between pages,** as well as add, close and rearrange the pages.

We sort of buried the lede: Switching between pages is what TabView spends most of its time doing!

Reply:
Do Tabs host pages?  If not, it may be an idea not to confuse these content panels with actual pages or frames?

Reply:
I was using "pages" in a casual sense. Would "switch between content" be better?

Reply:
Content sounds more accurate to me